examples/program_message_from_aleph.json

```diff
@@ -14,7 +14,7 @@
         "allow_amend": false,
         "code": {
             "encoding": "squashfs",
-            "entrypoint": "main:app",
+            "entrypoint": "run.sh",
```

Comment:
Modify this endpoint on that file is generic for all the examples. So maybe instead modify it, should be better to create a new message example file with needed endpoint.

Reply:
sorry it was added by error, will remove it